src/main/webapp/mapScript.js

```diff
@@ -89,53 +96,66 @@ function addNewTableItem(name, index) {
   tabel.appendChild(newPlace);
 }
 
-function deletePlace(contentId) {
-  let tableItem = document.getElementById("place" + contentId);
+function getActualIndex(placeIndex) {
+  var actualIndex = 0;
+  for (var i = 0; i < markersArray.length; i++) {
+    if (markersArray[i].data.index == placeIndex) {
+      actualIndex = i;
+      break;
+    }
+  }
+  return actualIndex;
```

Comment:
This seems problematic if the item is not found, as it will return 0 which indicates it is the first item rather than not found.

Reply:
Done